meson_options.txt

```diff
@@ -8,6 +8,7 @@ option('introspection', type : 'boolean', value : true, description : 'generate
 option('lvfs', type : 'boolean', value : true, description : 'enable LVFS remotes')
 option('man', type : 'boolean', value : true, description : 'enable man pages')
 option('pkcs7', type : 'boolean', value : true, description : 'enable the PKCS7 verification support')
+option('kernel_safety_checks', type : 'boolean', value : true, description : 'enable kernel version safety checks')
```

Comment:
why an option?

Reply:
Is there anything we can do to test this at runtime? It could be we have another kernel issue that was possibly backported in the future, then we'd need `kernel_safety_checks2` for all the variations of RHEL 7.x, 8.x etc. I'm also not hugely enthusiastic about rebuilding fwupd in response to kernel backports happening.

Reply:
Hmm, okay. Could we move this into a config file rather than a build time option? It certainly is easier to get someone to change a config file setting (and to patch in a build) rather than getting them to recompile fwupd with it's signed builder requirement. A config file could also be a generic way to do this kind of requirement matching. I'm not sure we want `<require><id>org.linux` etc but a more generic way seems a sensible thing to think about.

Reply:
Also, i think `kernel_version_checks` is a better name.